OpenTokMeet/app/src/main/java/com/opentok/android/meet/Room.java

```diff
@@ -109,22 +116,19 @@ public void disconnect() {
         }
     }
 
-    private Publisher getCustomPublisher(){
-        return new Publisher(
-                mContext,
-                "Android",
-                mActivity.getCapturerResolutionPub(),
-                mActivity.getCapturerFpsPub()
-        );
+    private Publisher createPublisher() {
+        return (new Publisher.Builder(mContext))
+                    .name(mPublisherName)
+                    .resolution(mPublisherRes)
+                    .frameRate(mPublisherFps)
+                    .build();
     }
 
     @Override
     protected void onConnected() {
         //check simulcast case for publisher
 
-        //mPublisher = new Publisher(mContext, "Android");
-        mPublisher = getCustomPublisher();
-        mPublisher.setName(mPublisherName);
+        mPublisher = createPublisher();
```

Comment:
You did not set the publisher setting anywhere before creating the publisher so the setting set in the home activity are not set. Please address that. The `setPublisherSettings()` method is never called. Sorry I noticed this now.